dev/integration_tests/gradle_deprecated_settings/README.md

```diff
@@ -1,6 +1,6 @@
 # Deprecated settings.gradle
 
-This project is meant to test that apps using the current `android/settings.gradle`
+This project is meant to test those apps using the current `android/settings.gradle`
```

Comment:
It's testing "that" certain apps can be still be built. Original was correct.

```suggestion
This project is meant to test that apps using the current `android/settings.gradle`
```

Reply:
I don't get it. that is is for singular and apps is plural. correct me If I am wrong

Reply:
you are correct, in that "that" is for singular, however, in this sentence "that" does not refer to the "apps", but to the fact of whether or not those apps can be built.

Reply:
Ah okay